readme.md

```diff
@@ -6,7 +6,7 @@ Nix package set and NixOS module for using COSMIC from NixOS. This is a temporar
 
 ### Flakes
 
-If you have an existing `configuration.nix`, you can use the `nixos-cosmic` flake with the following:
+If you have an existing `configuration.nix`, you can use the `nixos-cosmic` flake by adding the following to /etc/nixos/flake.nix:
```

Comment:
```suggestion
If you have an existing `configuration.nix`, you can use the `nixos-cosmic` flake with the following in an adjacent `flake.nix`:

**Note:** If switching from traditional evaluation to flakes, `nix-channel` will no longer have any effect on the nixpkgs your system is built with, and therefore `nixos-rebuild --upgrade` will also no longer have any effect. You will need to use `nix flake update` from your flake directory to update nixpkgs and nixos-cosmic.
```

Reply:
Is this suggestion acceptable? If so, I'll commit it to this branch and then merge this PR

Reply:
I've gone ahead and applied this suggestion. If it's sufficient feel free to close this PR and if not, let me know and we can come up with something that works for you :)